book/src/workspace-structure.md

```diff
@@ -0,0 +1,123 @@
+# Workspace Structure
+
+All dist projects, regardless of how many packages they may contain, are
+considered workspaces. A minimalist dist project would be a workspace that
+contains a single package.
+
+Dist has two types of configuration:
+
+- **Workspace**:
+- **Package**: 
+
+The vast majority of configuration for dist is workspace configuration. Because
+we understand language-specific config manifests like `Cargo.toml` and
+`package.json`, package configuration ususally does not require any additional
+files.
+
+## Workspace configuration
+
+A workspace file may be:
+
+* `dist-workspace.toml`
+* (deprecated in 0.23.0) `Cargo.toml` (Rust users only, goes in `[workspace.metadata.dist]`)
+
+### Specifying The Members Of Your Workspace
+
+Create a `workspace.members` field in your `dist-workspace.toml` that points to
```

Comment:
I wonder if we want to link to a live example of this kind of repo? https://github.com/axodotdev/axolotlsay-hybrid